common/fileview2/fvtransform.cpp

```diff
@@ -637,11 +637,11 @@ unsigned MappingParser::lexToken()
                 cur++;
             tokenType = TokId;
         }
-        else if (isdigit(next) ||
-                 ((next == '-') && (cur+1 < lenInput) && isdigit(buffer[cur+1])))
+        else if (isdigit_char(next) ||
+                 ((next == '-') && (cur+1 < lenInput) && isdigit_char(buffer[cur+1])))
         {
             cur++;
-            while ((cur < lenInput) && (isdigit(buffer[cur])))
+            while ((cur < lenInput) && (isdigit_char(buffer[cur])))
```

Comment:
In the interests of making it possible to review this change and get it merged without overwhelming reviewers, I would recommend breaking it into smaller changes each of which can be reviewed and verified independently. For example, a commit that contained nothing but a global rename of these isXXX functions would be very easy to get accepted and merged

Reply:
Good suggestion. Will try